samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1430,6 +1430,108 @@
           "Resource": "*"
         }]
       }
+    },
+    "DecodeSSMParameterPolicy": {
+      "Description": "Gives access to SSM key and parameter to load secrets in this account.",
+      "Parameters": {
+        "ParameterName": {
+          "Description":"The name of the secret stored in SSM in your account."
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ssm:DescribeParameters"
+            ],
+            "Resource": "*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "ssm:GetParameters",
+              "ssm:GetParameter"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "arn:${AWS::Partition}:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${parameterName}",
+                {
+                  "parameterName": {
+                    "Ref": "ParameterName"
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "kms:Decrypt"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "arn:${AWS::Partition}:ssm:${AWS::Region}:${AWS::AccountId}:alias/aws/ssm"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "DecodeSSMParameterCustomKeyPolicy": {
```

Comment:
We don't need this policy template. Let's just use this PR to define the `SSMParameterReadPolicy` for SSM API permissions. Then, if users use a custom KMS key to encrypt a parameter, they can add the existing `KMSDecryptPolicy` to give permissions to use that custom key for decrypting the SSM parameter value.